importanize/formatters.py

```diff
@@ -0,0 +1,139 @@
+# -*- coding: utf-8 -*-
+from __future__ import print_function, unicode_literals
+
+import itertools
+import operator
+import re
+
+
+class Formatter(object):
+    def __init__(self, statement):
+        self.statement = statement
+
+
+class IndentWithTabsFormatter(Formatter):
```

Comment:
Actually my changes are not finalised, I just pushed to check if tests were OK in Travis. **Thanks for the review**. I'll add docstrings when my formatters will be cleaner.